src/pcrlock/pcrlock.c

```diff
@@ -4504,6 +4504,7 @@ static int verb_make_policy(int argc, char *argv[], void *userdata) {
                                 srk_handle,
                                 encryption_session,
                                 &authnv_policy_digest,
+                                /* uses_pin= */ true,
```

Comment:
this is the only place where we call tpm2_seal_data, and at this place you pass the parameter unconditoinally as true, which has the effect of leaving TPMA_OBJECT_NODA off...

thus you are not actually changing anything here, at all!

Reply:
Correct. Although there's a comment on `tpm2_seal_data` noting that `tpm2_seal` should be refactored on top of it sometime in the future. So I thought this might be wanted so that no one forgets to include `noDA` then.

Reply:
I'll remove this part of the change.